tests/keras/layers/convolutional_test.py

```diff
@@ -933,5 +935,55 @@ def test_cropping_3d():
     with pytest.raises(ValueError):
         layer = convolutional.Cropping3D(cropping=lambda x: x)
 
+
+@pytest.mark.skipif(K.backend() != 'tensorflow', reason='Requires TF backend')
+@keras_test
+def test_depthwise_conv_2d():
```

Comment:
Do we still need this test at all? Shouldn't coverage be achieved in the `mobilenet` tests?

Reply:
@fchollet as you commented, there are redundant tests between backend tests and layer tests. I just wrote the both to fit the current testing format, but now a `test_depthwise_conv_2d ` in layer tests has been removed.